custom_dc/manage_repo.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: default
+title: Manage a repo
+nav_order: 5
+parent: Custom Data Commons
+---
+
+{:.no_toc}
+# Manage a repo
+
+* TOC
+{:toc}
+
+
+## Use a prebuilt image
+
+While you are just testing out data changes, you don't need to build the website, but can just use a prebuilt Data Commons image.
+
+Data Commons provides two prebuilt images in the Google Artifact Registry that you can download to run in a Docker container:
+
+- `gcr.io/datcom-ci/datacommons-website-compose:stable`. This is a tested, stable version but may be several weeks old.
+- `gcr.io/datcom-ci/datacommons-website-compose:latest`. This is the latest built version that is running in production.
+
+If you want to pick up the latest prebuilt version, run the following command from the `website` directory:
+
+```shell  
+docker pull gcr.io/datcom-ci/datacommons-website-compose:latest
+```
+Then, [restart Docker](quickstart.md#start-services), specifying that repo as the argument to the `docker run` command.
+
+## Build a local image {#build-repo}
+
+You will need to build a local image in any of the following cases:
+- You are making substantive changes to the website UI
+- You are ready to deploy your custom site to GCP
+
+Rather than building from "head", that is, the very latest changes in Github, which may not have been tested, we recommend that you use a tested "release", with the tag `customdc_stable`, available at [https://github.com/datacommonsorg/website/releases/tag/customdc_stable](https://github.com/datacommonsorg/website/releases/tag/customdc_stable).
+
+Note: If you are working on a large-scale customization, we recommend that you use a version control system to manage your code. We provide procedures for Github, and assume the following:
+- You have a Github account and project.
+- You have created a fork off the main Data Commons `website` repo (https://github.com/datacommonsorg/website) and a remote that points to it, and that you will push to that fork. 
+
+
+### Sync a local workspace to the stable release
```

Comment:
Thanks for figuring it out and documenting it!

Reply:
Thanks, it took me a while!